mne/channels/montage.py

```diff
@@ -1681,6 +1683,99 @@ def read_custom_montage(
     return montage
 
 
+@verbose
+def read_meg_montage(system, *, verbose=None):
+    """Load canonical MEG sensor definitions from CSV files.
+
+    Parameters
+    ----------
+    system : str
+        The MEG system name. Currently supported: 'neuromag', 'ctf151' or
+        'ctf275'.
+
+    Returns
+    -------
+    info : Info
+        An Info object containing the canonical sensor definitions.
+
+    Notes
+    -----
+    This function loads pre-defined canonical sensor positions and
+    orientations from CSV files stored in mne/channels/data/montages/.
+
+    .. versionadded:: 1.11
+    """
+    pd = _soft_import("pandas")
+
+    # Validate system input
+    _check_option("system", system, ["neuromag", "ctf151", "ctf275"])
+
+    # Map system names to CSV filenames
+    system_files = {
+        "neuromag": "neuromag306.csv",
+        "ctf151": "ctf151.csv",
+        "ctf275": "ctf275.csv",
+    }
+
+    # Load the CSV file
+    montage_dir = Path(__file__).parent / "data" / "montages"
+    csv_file = montage_dir / system_files[system]
+
+    if not csv_file.exists():
+        raise FileNotFoundError(f"Canonical sensor file not found: {csv_file}. ")
+
+    # Read sensor definitions
+    # TODO: Refactor not to use Pandas (should be able to parse without it)
+    df = pd.read_csv(csv_file)
```

Comment:
`pandas` is an optional dependency of MNE and not everyone will have it. Is it easy enough to just parse the CSV directly / manually? Usually we try to do that if it's not too much work